src/lib/toolchain_test.rs

```diff
@@ -1,46 +1,59 @@
 use super::*;
+use crate::test;
+
+#[test]
+#[should_panic]
+fn invalid_toolchain() {
+    wrap_command("invalid-chain", "true", &None);
+}
 
 #[test]
 fn wrap_command_none_args() {
-    let output = wrap_command("mychain", "testcommand", &None);
+    if test::is_not_rust_stable() {
+        let output = wrap_command("nightly", "true", &None);
```

Comment:
this breaks the build on beta (see travis/appveyor builds). lets change to envmnt::get_or_panic("CARGO_MAKE_RUST_CHANNEL")

Reply:
Thanks! This was the env var I wasn't aware of which is why I implemented this gross hack in the tests :)